web-gateway/src/main/java/controllers/Nav.java

```diff
@@ -79,13 +82,28 @@ public String formatInstant(Instant instant) {
     public String formatDuration(Duration duration) {
         Pair<ChronoUnit, Long> pair = Durations.fromJDuration(duration);
 
-        String key ;
-        switch (pair.first()){
-            case WEEKS:   {key = "durationWeeks"; break;}
-            case DAYS:    {key = "durationDays"; break;}
-            case HOURS:   {key = "durationHours"; break;}
-            case MINUTES: {key = "durationMinutes"; break;}
-            default:      {key = "durationSeconds"; break;}
+        String key;
+        switch (pair.first()) {
+            case WEEKS: {
+                key = "durationWeeks";
+                break;
+            }
+            case DAYS: {
+                key = "durationDays";
+                break;
+            }
+            case HOURS: {
+                key = "durationHours";
+                break;
+            }
+            case MINUTES: {
+                key = "durationMinutes";
+                break;
+            }
+            default: {
+                key = "durationSeconds";
+                break;
+            }
```

Comment:
Yes, please revert the change in this file, since it isn't part of the changes for this pull request.